core/test/models/workarea/fulfillment/item_test.rb

```diff
@@ -3,32 +3,36 @@
 module Workarea
   class Fulfillment
     class ItemTest < TestCase
```

Comment:
Something changed in Mongo where this stopped working in-memory (but works fine when things are persisted).